README.md

```diff
@@ -67,7 +67,6 @@ repo:
 
 # Add '@domain/core' label to any change within the 'core' package
 @domain/core:
-- package/core/*
```

Comment:
This line is not necessary and is actually confusing. We ended up with four config lines instead of two (before `dot`)

```
"area: packages":
  - "packages/*"
  - "packages/.*"
  - "packages/**/*"
  - "packages/**/.*"
```